src/components/TextInput/baseTextInputPropTypes.js

```diff
@@ -129,6 +132,7 @@ const defaultProps = {
     icon: null,
     shouldUseDefaultValue: false,
     multiline: false,
+    canInterceptSwipe: false,
```

Comment:
Boolean props should be prefixed with `should` or `is`.

Reply:
Updated. Pushed a new commit.